AlpsSDK/Repositories/Devices/BeaconTripleRepository.swift

```diff
@@ -14,13 +14,13 @@ let kBeaconFile = "kBeaconFile.Alps"
 final public class BeaconTripleRepository: AsyncReadable {
     
     typealias DataType = IBeaconTriple
-    public var items = [IBeaconTriple]() {
+    internal private(set) var items = [IBeaconTriple]() {
         didSet {
```

Comment:
Might have to open it else we cannot get the IBeaconTriples registered in the app ?